openc3-cosmos-init/plugins/packages/openc3-vue-common/src/components/Openc3Screen.vue

```diff
@@ -165,6 +166,20 @@
         opacity="0.8"
         absolute
         attach
+        scroll-strategy="none"
+      />
+      <v-btn
+        v-show="errors.length !== 0"
+        icon="mdi-alert"
+        variant="text"
+        density="compact"
+        data-test="error-graph-icon"
+        aria-label="Show Errors"
+        @click="
+          () => {
+            errorDialog = true
+          }
+        "
```

Comment:
Why the new button?

Reply:
It shows errors in "inline" mode (for Notebooks)